packages/themes/vite.config.ts

```diff
@@ -26,5 +26,17 @@ export default defineConfig({
       enabled: true,
       reporter: 'text',
     },
+    /**
```

Comment:
Do we need this again? I'm not sure why it was removed 🤔

Reply:
Geoff removed the vitest.config.ts and put it into the vite.config.ts in #4363, I just redid the changes in a separate PR. I’m fine with both approaches (two configs or one config).